variables.tf

```
@@ -17,6 +17,11 @@ variable "batch_size" {
   description = "Maximum number of records passed for a single Lambda invocation"
 }
 
+variable "enable_kinesis_mapping" {
+  description = "Determines if the event source mapping will be enbaled on creation"
```

```suggestion
  description = "Determines if the event source mapping will be enabled"
```
It is possible to disable existing mapping by changing this parameter to false.

---

👍